### ISSUE TITLE
Set-vRAReservationNetwork Issue

### ISSUE THREAD

**Issue by thecloudxpert** [jakkulabs/PowervRA#82]
I'm trying to use Set-vRAReservationNetwork to set the network's available within a reservation.  I dont currently have any network profiles setup as INFOBLOX is being used and this isn't available yet. I have 8 reservations and about 40 VLANs to add to each so i thought this would be a good thing to use.  As an example, without looping through the list of VLANs, I'm using:

Get-vRAReservation -Name "RES_Test-Cluster01_001" | Set-vRAReservationNetwork -NetworkPath "VLAN_10"

And it doesn't seem to work, neither does it work if i create a temporary Network Profile and use:

Get-vRAReservation -Name "RES_Test-Cluster01_001" | Set-vRAReservationNetwork -NetworkPath "VLAN_10" -NetworkProfile "Test"

Version: PowerCLI 6.3R1, PowerShell 4.0, PowerVRA 1.3.2

**Comment by chelnak** [comment]
Looks like the function was only initially set up to update the network profiles of selected networks.

I have fixed this now in branch **bugfix-set-vrareservationnetworkprofile**. 

It will be merged in to the development branch once a test has been written and @jonathanmedd has approved.

```
    .EXAMPLE
    Add a new network path to a reservation and assign a network profile

    Get-vRAReservation -Name "Reservation01" | Set-vRAReservationNetwork -NetworkPath "VM Network" -NetworkProfile "Test Profile 2"

    .EXAMPLE
    Update the network profile of an existing network path

    Get-vRAReservation -Name "Reservation01" | Set-vRAReservationNetwork -NetworkPath "VM Network" -NetworkProfile "Test Profile 1"

    .EXAMPLE
    Add a new network path to a reservation

    Get-vRAReservation -Name "Reservation01" | Set-vRAReservationNetwork -NetworkPath "Test Network"
```

**Comment by chelnak** [comment]
Fixed in v1.4.0
